scrape.js

```diff
@@ -165,6 +171,30 @@ const setupCourse = (currentSection, currentSectionInfo, parsedLine) => {
   currentSection.course.comments = comments
   currentSectionInfo.comments = []
   currentSection.section = { enrollment: parsedLine.enrollment, intervals: [] }
+
+  if (parsedLine.course.abbreviation !== "" && parsedLine.course.number !== "")
+    Course.where({
+      "abbreviation": parsedLine.course.abbreviation,
+      "number": parsedLine.course.number,
+    })
+    .fetch()
+    .then(course => {
+      if (course !== null)
+        console.log("Course already setup: " + parsedLine.course.abbreviation + " " + parsedLine.course.number)
+      else
+        new Course({
+          "abbreviation": parsedLine.course.abbreviation,
+          "number": parsedLine.course.number,
+          "hours": parsedLine.course.hours,
+          "specialEnrollment": parsedLine.course.specialEnrollment,
+          "isWebBased": parsedLine.course.isWebBased,
+          "isComIntensive": parsedLine.course.isComIntensive,
+        })
+        .save()
+        .then(newCourse => {
+          console.log("     New course setup: " + parsedLine.course.abbreviation + " " + parsedLine.course.number)
+        })
+      })
```

Comment:
This is just a demonstration for creating Course records.

They have no dependencies, so it is a very simple example.

This will be removed before merge since we intend to move database logic elsewhere.